guides/v3.3.0/configuring-ember/debugging.md

```diff
@@ -130,7 +130,7 @@ RSVP.on('error', function(error) {
 #### Errors within `Ember.run.later` ([Backburner.js](https://github.com/ebryn/backburner.js))
 
 Backburner has support for stitching the stacktraces together so that you can
-track down where an erroring `Ember.run.later` is being initiated from. Unfortunately,
+track down where an error thrown by `Ember.run.later` is being initiated from. Unfortunately,
```

Comment:
Happy to change this one if people disagree but I don't think `erroring` is technically a word.